app/views/users/edit.html.erb

```diff
@@ -0,0 +1,16 @@
+<% breadcrumb :user, @person %>
+
+<%= simple_form_for([@person, @user]) do |f| %>
+  <div class="card">
+    <div class="card-header">
+      <h1>Editing user information</h1>
+    </div>
+    <div class="card-body">
+      <%= render "form", f: f %>
+    </div>
+    <div class="card-footer">
+      <%= link_to "Cancel", @person, class: 'btn btn-warning' %>
+      <%= f.button :submit, class: 'btn btn-primary' %>
```

Comment:
create a new issue for this.. when i change my password it says "you must be logged in to do that". @braddoeswebdev i know that seems like a devise thing, but is it possible to change the tag when you change your password?

![image](https://github.com/user-attachments/assets/58a5a0f9-b975-4d87-9815-50d4ce0a6539)

Reply:
Your sessions all get invalidated when you change your password, so the User#update action either needs to redirect to the login page with a more appropriate flash message or re-login with the new password.